src/store/holochain.js

```diff
@@ -285,14 +301,16 @@ export default {
       state.reconnectingIn = -1
       state.firstConnect = false
     },
-    setHoloClientAndDnaAlias (state, payload) {
-      const { holoClient, dnaAlias } = payload
-      console.log('HOLO DNA ALIAS : ', dnaAlias)
-      state.dnaAlias = dnaAlias
-      state.holoClient = holoClient
+    setHoloClient (state, payload) {
+      console.log('holo client: ', payload)
```

Comment:
remove log

Reply:
This log was previously there - I just updated the value. Do we no longer want to show this?

Reply:
I don't see this one in the previous code. I don't think this is going to be useful day to day